requirements.txt

```diff
@@ -40,6 +40,7 @@ django~=2.0.0
 gunicorn==19.9.0
 mailchimp3==3.0.7
 mistune==0.8.4
+more-itertools==7.2.0
```

Comment:
Still needed?

Reply:
Yea, I'm using `collapse` in the tests to avoid duplicating the logic in get_files.